src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/haflows/HaFlowHistorySpec.groovy

```diff
@@ -244,4 +280,20 @@ class HaFlowHistorySpec extends HealthCheckSpecification {
                     vlanId: it.endpointVlan, innerVlanId: it.endpointInnerVlan)
         }
     }
+
+    void verifyDumpSection(HaFlowDumpPayload dump, HaFlowExtended haFlow) {
```

Comment:
Ok. Now when you have this comparison in several places, it's time to move it to either 'HaFlowExtended' or 'HaFlowDumpPayload'.

Reply:
It seems like it is more history-related, let's keep it in the HaFlowHistorySpec.

Reply:
It's a bad practice to keep some helper methods in spec class. It is hard to imagine a scenario where developer remembers all the helpers methods from classes and successfully reuses the ones.
On the other hand, if you put it into HaFlowHistoryEvent class, it will be auto-suggested even in other specs.
If you don't want to refactor now, at least leave TODO with link to github issue.

Reply:
updated.